### ISSUE TITLE
Mod not able to fetch heads.

### ISSUE THREAD

**Issue by WERooo** [mega12345mega/NBT-Editor#11]
mod version: 1.0.6
On fresh MultiMC instance with 
 Fabric API 0.55.3
 Fabric Loader 0.14.7

causes that error:
```
[22:05:11] [Auto Head Refresh/ERROR]: Failed to update database! Check above for any errors.
[22:05:16] [Auto Head Refresh/ERROR]: Failed to fetch heads (no-cache) | Stack Trace:
[22:05:16] [Auto Head Refresh/ERROR]: java.io.IOException: Server returned HTTP response code: 503 for URL: https://minecraft-heads.com/scripts/api.php?cat=alphabet&tags=true
[22:05:16] [Auto Head Refresh/INFO]: Attempting fallback provider for: alphabet
```

Full log:
https://pastebin.com/tcFmwrDe

**Comment by mega12345mega** [comment]
Unfortunately, the head database website ([minecraft-heads.com](minecraft-heads.com)) has issues sometimes, which causes it to reject requests.
While there isn't anything I can do about it, I have implemented code to use a fallback provider.
However, I messed up the code, so the fallback provider response is discarded (I was unable to properly test it, as the database was working fine while I developed the feature).

It's currently on my todo list to fix!
Hopefully the database starts working again, but in the meantime, you can use the website directly.
If you don't have OP, you can replace the /give part of the command from the website with /get item. (This will also allow you to run give commands longer than the normal limit, if you enable it in the /nbteditor config.)

**Comment by mega12345mega** [comment]
I just rolled out the fix in the latest version (v1.6.1)!
Sorry for taking so long.
